README.md

```diff
@@ -477,19 +477,46 @@ runningCluster, err := w.Clusters.CreateAndWait(ctx, clusters.CreateCluster{
 })
 ```
 
-## `io.Reader` integration for DBFS
+## Integration with `io` interfaces for DBFS
```

Comment:
please also update the reference in table of contents in the top of the file, otherwise we have a broken link

Reply:
Done.